src/makeInfinityAdder.js

```diff
@@ -4,7 +4,32 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let totalSum = 0;
+  let currentSum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
+      const result = totalSum + currentSum;
+
+      totalSum = 0;
+      currentSum = 0;
```

Comment:
Since `totalSum` is not needed, these lines should be removed along with the `totalSum` variable.